src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -2543,51 +2312,39 @@ namespace WaterToAirHeatPumpSimple {
-            ratioVL = (state.dataWaterToAirHeatPumpSimple->LoadSideMassFlowRate /
-                       (AirVolFlowRateRated * PsyRhoAirFnPbTdbW(state,
-                                                                state.dataEnvrn->StdBaroPress,
-                                                                state.dataWaterToAirHeatPumpSimple->LoadSideInletDBTemp,
-                                                                state.dataWaterToAirHeatPumpSimple->LoadSideInletHumRat,
-                                                                RoutineName)));
-
-            if (state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).DesignWaterMassFlowRate > 0.0) {
-                ratioVS = (state.dataWaterToAirHeatPumpSimple->SourceSideMassFlowRate) /
-                          (state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).DesignWaterMassFlowRate);
+            ratioVL = (LoadSideFullMassFlowRate /
+                       (AirVolFlowRateRated * Psychrometrics::PsyRhoAirFnPbTdbW(state,
+                                                                                state.dataEnvrn->StdBaroPress,
+                                                                                state.dataWaterToAirHeatPumpSimple->LoadSideInletDBTemp,
+                                                                                state.dataWaterToAirHeatPumpSimple->LoadSideInletHumRat,
+                                                                                RoutineName)));
+
+            if (simpleWatertoAirHP.DesignWaterMassFlowRate > 0.0) {
+                ratioVS = (state.dataWaterToAirHeatPumpSimple->SourceSideMassFlowRate) / (simpleWatertoAirHP.DesignWaterMassFlowRate);
             } else {
                 ratioVS = 0.0;
             }
 
-            state.dataWaterToAirHeatPumpSimple->QLoadTotal =
-                TotalCapRated *
-                CurveValue(
-                    state, state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).TotalCoolCapCurveIndex, ratioTWB, ratioTS, ratioVL, ratioVS);
-            state.dataWaterToAirHeatPumpSimple->QSensible =
-                SensCapRated * CurveValue(state,
-                                          state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).SensCoolCapCurveIndex,
-                                          ratioTDB,
-                                          ratioTWB,
-                                          ratioTS,
-                                          ratioVL,
-                                          ratioVS);
+            simpleWatertoAirHP.QLoadTotal =
+                TotalCapRated * CurveManager::CurveValue(state, simpleWatertoAirHP.TotalCoolCapCurveIndex, ratioTWB, ratioTS, ratioVL, ratioVS);
+            simpleWatertoAirHP.QSensible =
+                SensCapRated *
+                CurveManager::CurveValue(state, simpleWatertoAirHP.SensCoolCapCurveIndex, ratioTDB, ratioTWB, ratioTS, ratioVL, ratioVS);
```

Comment:
QLoadTotal and QSensible are the full load coil outputs based on performance curves used for this coil model.